README.md

```diff
@@ -187,6 +187,67 @@ result = my_zosmf.get_info()
 ```
 The result will be a JSON object containing z/OSMF information
 
+## Local setup for macOs
+
+Make sure that the current python version is `3.10` or older
+
+If it is in an older version , you can downgrade to a lower versions by following the below steps:
+
+```bash
+brew install pyenv
+
+echo 'PATH=$(pyenv root)/shims:$PATH' >> ~/.zshrc
+
+pyenv install 3.10 ## Example for python 3.10 version
+```
+
+After this , you can clone the repository
+
+```
+git clone https://github.com/zowe/zowe-client-python-sdk.git
+
+```
+
+You can create a remote <origin> for it
+    
+```
+git remote add <originname> https://github.com/zowe/zowe-client-python-sdk.git
+
+```
+
+Checkout for the next branch
+    
+```
+git fetch <origin-name> && git checkout next
+    
+```
+
+This is for using the python 3.10 or lower versions for your local setup
+
+```
+pyenv local <version> ## you can get this by running pyenv versions
+
+## This will create a python-version file , this version will only be used locally for this project
+
+```
+
+Create a virtual environment
+
+```
+
+python -m venv <envname>
```

Comment:
At the beginning of these steps, it is noted that "the current python version is `3.10` or older"..

However, `venv` will not work on Python 3.1 and 3.2. Maybe we should consider mentioning a minimum Python3 version, or add a note saying that `venv` is only supported for Python 3.3 and above.